Composite/Plugins/Functions/FunctionProviders/StandardFunctionProvider/Pages/GetPageIdFunction.cs

```diff
@@ -24,7 +24,7 @@ public GetPageIdFunction(EntityTokenFactory entityTokenFactory)
 
         public override object Execute(ParameterList parameters, FunctionContextContainer context)
         {
-            if (parameters.TryGetParameter<SitemapScope>("SitemapScope", out var sitemapScope) == false)
+            if (parameters.TryGetParameter<SitemapScope>(nameof(SitemapScope), out var sitemapScope) == false)

```

Comment:
`if (!parameters.TryGetParameter<SitemapScope>(nameof(SitemapScope), out var sitemapScope))`

?

Reply:
I didn't notice this when was updating this line, thanks, changed